HW_TypeScript_Type_annotation_and_primitive_types.ts

```diff
@@ -0,0 +1,116 @@
+class School {
+    directions: any = [];
+
+    addDirection(direction: any) {
+        this.directions.push(direction);
+    }
+}
+
+class Direction {
+    levels: any = [];
+
+    get name(): any {
+        return this._name;
+    }
+
+    constructor(name: string) {
+        this._name = name;
+    }
+
+    addLevel(level: any) {
+        this.levels.push(level);
+    }
+}
+
+class Level {
+    groups: any = [];
+
+    constructor(name: string, program: any) {
+        this.name = name;
+        this._program = program;
+    }
+
+    get name(): string {
+        return this._name;
+    }
+
+    get program(): any {
+        return this._program;
+    }
+
+    addGroup(group: any) {
+        this.groups.push(group);
+    }
+}
+
+class Group {
+    _students: any = [];
+
+    get students(): any {
+        return this._students;
+    }
+
+    constructor(directionName: string, levelName: string) {
+        this.directionName = directionName;
+        this.levelName = levelName;
+    }
+
+    addStudent(student: any) {
```

Comment:
Please do not forget to add explicit type annotation for the return function value.